shells/pipes-shell/README.md

```diff
@@ -20,9 +20,32 @@ Also inside each platform folder is a deploy folder that contains tools for buil
 
 `pack-stats.json` contains Webpack information that can be used in tools like [(Online) Webpack Visualizer](https://chrisbateman.github.io/webpack-visualizer/).
 
-### Flags
+## Usage
 
-pipes-shell/web/index.html?log[=[0..2]]&remote-explore-key=[key]
+pipes-shell exposes JS entry/exit points designed to be bound into another process. These entry points can also be exercised via console. Here are some examples:
+
+```
+> ShellApi.observeEntity(`{"type": "address", "name": "East Mumbleton"}`)
+> ShellApi.receiveEntity(`{"type": "com.google.android.apps.maps"}`)
+> ShellApi.receiveEntity(`{"type": "com.music.spotify"}`)
+```
+Results are returned via `DeviceClient.foundSuggestions(json)` (if it exists)
```

Comment:
Give example actual json that might be passed back? Or, explain where a reader could look to learn more about it.

Reply:
Done, thanks for the proof-read.